.circleci/config.yml

```diff
@@ -86,15 +86,17 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/docs
-      # Create a folder for results to live in
-      - run: "[ -d lint-results ] || mkdir lint-results"
+      - run:
+          name: Create a folder for results to live in
+          command: "[ -d build/lint-results ] || mkdir -p build/lint-results"
```

Comment:
might as well place under `./build` which is git-ignored and where other test results are saved